docs/web3-eth-accounts.rst

```diff
@@ -8,19 +8,15 @@ web3.eth.accounts
 
 The ``web3.eth.accounts`` contains functions to generate Ethereum accounts and sign transactions and data.
 
-.. note:: This package has NOT been audited and might potentially be unsafe. Take precautions to clear memory properly, store the private keys safely, and test transaction receiving and sending functionality properly before using in production!
-
-To use this package standalone use:
-
+.. note:: This package got NOT audited until now. Take precautions to clear memory properly, store the private keys safely, and test transaction receiving and sending functionality properly before using in production!
```

Comment:
This is crucial. If you mean that the package is NOT audited, use the phrasing below. The current phrasing implies that it HAS been audited!

This package has not been audited. Take precautions to clear memory properly, store the private keys safely, and test transaction receiving and sending functionality properly before using in production!